test/runtests.jl

```diff
@@ -24,6 +24,18 @@ function test_manifold(M::Manifold, pts::AbstractVector)
     @test norm(M, pts[1], tv1) ≈ sqrt(dot(M, pts[1], tv1, tv1))
 end
 
+function text_arraymanifold()
```

Comment:
`text` should be `test`

Reply:
I did not even read that yet, since I was really confused by the tangent vector not being orthogonal.